src/ncp/spinel.h

```diff
@@ -529,6 +529,7 @@ typedef enum
     SPINEL_PROP_PHY_CCA_THRESHOLD       = SPINEL_PROP_PHY__BEGIN + 4, ///< dBm [c]
     SPINEL_PROP_PHY_TX_POWER            = SPINEL_PROP_PHY__BEGIN + 5, ///< [c]
     SPINEL_PROP_PHY_RSSI                = SPINEL_PROP_PHY__BEGIN + 6, ///< dBm [c]
+    SPINEL_PROP_PHY_SENSITIVITY         = SPINEL_PROP_PHY__BEGIN + 7, ///< dBm [c]
```

Comment:
Can you please update the spinel documentation at `spinel.prop-phy.md' --> https://github.com/openthread/openthread/blob/master/doc/spinel-protocol-src/spinel-prop-phy.md